src/api/api.cpp

```diff
@@ -1522,6 +1522,21 @@ extern "C" migraphx_status migraphx_module_add_instruction(migraphx_instruction_
     return api_error_result;
 }
 
+extern "C" migraphx_status migraphx_module_get_last_instruction(migraphx_instruction_t* out,
```

Comment:
We can remove these changes (commits) which are related to exposing this API part.

Reply:
Removed them